src/style.css

```diff
@@ -1 +1,51 @@
-/* add styles here */
+@font-face {
+  src: url(../src/fonts/Avenir.ttc);
+  font-family: Avenir, sans-serif;
```

Comment:
There's a typo in the font-family declaration. It should be `font-family: 'Avenir', sans-serif;` to ensure proper font usage.